Sources/DangerShellExecutor/ShellExecutor.swift

```diff
@@ -61,9 +61,11 @@ public struct ShellExecutor: ShellExecuting {
         let pipe = Pipe()
         task.standardOutput = pipe
         task.launch()
-        task.waitUntilExit()
 
         let data = pipe.fileHandleForReading.readDataToEndOfFile()
+
+        task.waitUntilExit()
```

Comment:
Would you still receive all the text output coming from the executed shell command after this change?
Given we are reading before the task ends

Reply:
We're calling `readDataToEndOfFile()` so we'll get the entire output of the command from the pipe before continuing to `task.waitUntilExit()`, which I assume will basically immediately complete at that point.

Reply:
@f-meloni does that seem correct?

Reply:
Let's give it a chance :)